api/v1/models/endpoint_change_request.go

```diff
@@ -40,6 +40,11 @@ type EndpointChangeRequest struct {
 	// ID of datapath tail call map
 	DatapathMapID int64 `json:"datapath-map-id,omitempty"`
 
+	// Disables lookup using legacy endpoint identifiers (container name,
+	// container id, pod name) for this endpoint
+	//
```

Comment:
nit: extra comment line here

Reply:
Took me a while to figure this out, since it's auto generated. For future reference, the trick is to use ` >-` in openapi.yaml for the description